classes/DepositPackage.inc.php

```diff
@@ -647,7 +647,7 @@ public function updateDepositStatus() {
 			case 'received':
 				$this->_deposit->setLockssReceivedStatus();
 				break;
-			case 'syncing':
+			case 'inProgress':
```

Comment:
I was going to leave both, but I couldn't find any signal of the `syncing`, the same applies for the `received` state above (https://github.com/SFULibrary/lockss-o-matic/blob/e7be1e24159cfa1b8776dfbce46021476d71a1fc/src/LOCKSSOMatic/SwordBundle/Controller/SwordController.php#L391-L394).

Reply:
I'd be inclined to leave the extra states, but add a comment that they're unknown/deprecated:
```
case 'syncing': // WARNING: The usage of 'syncing' is unknown and it may be removed in the future!
case 'inProgress':
    ...
```

Reply:
Done.